File: BondPrototype/Models/Person.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BondPrototype.Models;

public class Person
{
    
    [Key] public int Id { get; set; }
    public string Name { get; set; }
    public DateTime? DateOfBirth { get; set; }
    public string Picture { get; set; }
    public string Biography { get; set; }

    public List<Movie> Directed { get; set; }
    public List<Movie> ActedIn { get; set; }
}

File: BondPrototype/Angular/src/app/api/models/person.ts
import {Movie} from './movie';
import {key} from '../actions/decorators';


export class Person {
    
    @key() public id: number;
    public name: string;
    public dateOfBirth: Date | null;
    public picture: string;
    public biography: string;
    public directed: Movie[];
    public actedIn: Movie[];
}